src/datastore_mad/remotes/libfs.sh

```diff
@@ -65,7 +65,7 @@ function image_format {
 #   @return string representation of the format, empty if error
 #-------------------------------------------------------------------------------
 function image_vsize {
-    echo "$($QEMU_IMG info "${1}" 2>/dev/null | sed -n 's/.*(\([0-9]*\) bytes).*/\1/p')"
+    echo "$($QEMU_IMG info --output json "${1}" | jq '."virtual-size"')"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Useless echo? Instead of &#39;echo $(cmd)&#39;, just use &#39;cmd&#39;.